app/models/solution.rb

```diff
@@ -116,6 +116,12 @@ def published_iterations
     iterations.not_deleted
   end
 
+  def up_to_date_tests_status
+    return if published_iterations.blank?
+
+    published_iterations.last.submission.test_runs.find_by(git_sha: exercise.git_sha)&.tests_status
+  end
```

Comment:
This will be the method that we use in the search index. We need to work out how we know to trigger the update.